master/getting-started/docker/installation/vagrant-coreos/index.md

```diff
@@ -64,24 +64,24 @@ And finally check that Docker is running on both hosts by running
 ## 2. Install Calico
 
 With your VMs running, and connectivity between them established,
-it is time to launch `calico/node`.
+it is time to launch `{{site.nodecontainer}}`.
 
-The Vagrant machines already have `calicoctl` installed. Use it to launch `calico/node`:
+The Vagrant machines already have `calicoctl` installed. Use it to launch `{{site.nodecontainer}}`:
 
     sudo ETCD_ENDPOINTS=http://172.17.8.101:2379 calicoctl node run --node-image={{site.imageNames["node"]}}:{{site.data.versions[page.version].first.title}}
 
 Append the `--use-docker-networking-container-labels` flag to the `calicoctl node run` command if you're combining
 [Docker Labels and {{site.prodname}} Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/security-using-docker-labels-and-calico-policy).
 
-Check that the `calico/node` container is running on this host:
+Check that the `{{site.nodecontainer}}` container is running on this host:
```

Comment:
I think this should be `{{site.noderunning}}`, which is what we are checking for?